lib/content_encoding.c

```diff
@@ -835,8 +835,8 @@ static const struct Curl_cwtype identity_encoding = {
 };
 
 
-/* supported content encodings table. */
-static const struct Curl_cwtype * const encodings[] = {
+/* supported general content decoders. */
+static const struct Curl_cwtype * const general_decoders[] = {
```

Comment:
When i wrote this, I preferred using "unencode" rather than "decode" to emphasize the context of encoding handling. So I'm not a big fan of the renaming.

Reply:
I can change that back. I though "decode" was the opposite of "encode", but English is not my first language.

Reply:
It is. "unencode" was always very weird to read in my eyes. What does it mean? In English you encode one way and you decode the other way.

Reply:
>  I thought "decode" was the opposite of "encode"

Yes it is indeed. And the word "unencode" is not very elegant I admit.
This was just because we are dealing with **encodings**.
Thus I may be ok with _decoding_ if you change it everywhare for consistency!

Reply:
Renamed as proposed.